src/components/Book.js

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import '../App.css';
 
 function Book(props) {
   const { book } = props;
+  const { remove } = props;
+  console.log(remove);//eslint-disable-line
```

Comment:
- [ ] It would be better if we remove the console log statements, once we are done with debugging, we should remove them to keep the code clean. Also, kindly avoid disabling inline rules, they are almost the same as changing the config files